pkgs/applications/misc/openambit/default.nix

```
@@ -0,0 +1,44 @@
+{ stdenv
+, cmake
+, fetchFromGitHub
+, libusb1
+, makeWrapper
+, python3
+, qtbase
+, qttools
+, udev
+, wrapQtAppsHook
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "openambit";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "openambitproject";
+    repo = pname;
+    rev = version;
+    sha256 = "1074kvkamwnlkwdajsw1799wddcfkjh2ay6l842r0s4cvrxrai85";
+  };
+
+  nativeBuildInputs = [ cmake makeWrapper qttools wrapQtAppsHook ];
```

```suggestion
  nativeBuildInputs = [ cmake qttools  ];
```
Not necessary if use qt5's mkDerivation. See https://nixos.org/nixpkgs/manual/#sec-language-qt